packages/next/README.md

```diff
@@ -5,9 +5,11 @@
       <img src="https://assets.vercel.com/image/upload/v1662130559/nextjs/Icon_light_background.png" height="128">
     </picture>
     <h1 align="center">Next.js</h1>
+    <p align="center">The React Framework</p>
```

Comment:
```suggestion
```
I think this is fine not including this. I think it may look funky with it below.

![CleanShot 2024-01-16 at 17 01 37@2x](https://github.com/vercel/next.js/assets/28912696/a8ca6c18-7499-491e-b3ed-d4febdc4f24a)